docs/hyperchains_setup.md

```diff
@@ -65,17 +64,16 @@ To ensure data persistence across container restarts:
 docker run -it --rm \
   -p 3013:3013 \
   -p 4000:4000 \
-  -e AETERNITY_CONFIG=/home/aeternity/aeternity.yaml \
-  -v ${PWD}/${NAME}/nodeConfig/aeternity.yaml:/home/aeternity/aeternity.yaml \
-  -v ${PWD}/${NAME}/nodeConfig/${NAME}_accounts.json:/home/aeternity/node/local/rel/aeternity/data/mnesia/${NAME}_accounts.json \
-  -v ${PWD}/${NAME}/nodeConfig/${NAME}_contracts.json:/home/aeternity/node/local/rel/aeternity/data/mnesia/${NAME}_contracts.json \
-  -v ${PWD}/data/mnesia:/home/aeternity/node/local/rel/aeternity/data/mnesia \
-  -v ${PWD}/data/mdw.db:/home/aeternity/node/local/rel/aeternity/data/mdw.db \
+  -v ${PWD}/${NAME}/nodeConfig/aeternity.yaml:/home/aeternity/.aeternity/aeternity/aeternity.yaml \
+  -v ${PWD}/${NAME}/nodeConfig/${NAME}_accounts.json:/home/aeternity/node/data/mnesia/${NAME}_accounts.json \
+  -v ${PWD}/${NAME}/nodeConfig/${NAME}_contracts.json:/home/aeternity/node/data/mnesia/${NAME}_contracts.json \
+  -v ${PWD}/data/mnesia:/home/aeternity/node/data/mnesia \
+  -v ${PWD}/data/mdw.db:/home/aeternity/node/data/mdw.db \
```

Comment:
The change works but as this is for persistent running, to work across restarts, it could run with a named contained (without the `-rm`). It would help a lot on the restart to avoid mapping all volumes again, specially to avoid conflict on the configs that was chosen for starting and should not be changed.